### PR TITLE
Add comptia certs

### PR DESCRIPTION
**Pull request by thegeekkid**
Shame on them!  They even made such a big deal out of 2fa on the Security+!  ;)

### CODE REVIEW

**Comment by Jawshy** [comment]
@thegeekkid, would you like to use one of these icons instead? 
1. Their logo: 
   comptia 
2. Their site's favicon: 
   comptia-2 

Closes https://github.com/2factorauth/twofactorauth/issues/2038

File: _data/other.yml [hunk 1]
```diff
@@ -103,6 +103,13 @@ websites:
       img: commuterbenefits.png
       tfa: No
       twitter: CommuterBenefit
+
```
Comment (Jawshy) [inline _data/other.yml]: @thegeekkid, could you remove those blank spaces?

Comment (thegeekkid) [reply _data/other.yml]: Ahh.  Sorry about that.  Stupid Notepad++.  Should have used vim.  ;)

Pushing that and the updated icon (I'll use the favicon).  Thanks for those!

**Review by stephengroat — APPROVED** [review]

File: _data/other.yml (hunk 1 above)
Comment (Carlgo11) [reply _data/other.yml]: @thegeekkid if you use notepad++ you can install the editorconfig plugin. That way your edits will automatically comply with our mark-up guidelines. :wink: 
https://github.com/editorconfig/editorconfig-notepad-plus-plus#install-from-the-plugin-manager

**Comment by Carlgo11** [comment]
I think Comptia is better suited for the education category.

**Review by stephengroat — CHANGES_REQUESTED** [review]
move to `education` catagory

File: _data/other.yml (hunk 1 above)
Comment (thegeekkid) [reply _data/other.yml]: Hmm.  Good to know.  ;)

**Comment by thegeekkid** [comment]
Ya... I was debating which to put it under when I was adding it.  I'll change it over.  :)

**Comment by thegeekkid** [comment]
Ok, moved it over.

**Review by stephengroat — APPROVED** [review]

**Pull request merged**